alignak/objects/config.py

```diff
@@ -1068,6 +1068,7 @@ def read_config_buf(self, buf):  # pylint: disable=R0912
         lines = buf.split('\n')
         line_nb = 0  # Keep the line number for the file path
         for line in lines:
+            filefrom = ''
```

Comment:
I will revert this modification because it breaks some tests in test_config. The importing filename is missing ...